v2/contribute/decisions/session/0023-jwks-endpoint.mdx

```diff
@@ -0,0 +1,32 @@
+---
+id: "0023"
+title: Add new JWKs endpoint on the standard path
+hide_title: true
+---
+
+import DecisionHeader from "/src/components/decisionLogs/DecisionHeader"
+import ArgumentPro from "/src/components/decisionLogs/ArgumentPro"
+import ArgumentCon from "/src/components/decisionLogs/ArgumentCon"
+import ArgumentNeut from "/src/components/decisionLogs/ArgumentNeut"
+
+# Add new JWKs endpoint on the standard path
+
+<DecisionHeader status="proposed" lastUpdate="2022-12-06" created="2022-12-06" deciders={["rishabhpoddar", "porcellus"]} proposedBy={["porcellus"]} />
+
+## Context and Problem Statement
+
+Have already have an jwks endpoint available in the Core, but it's exposed under a non-standard path.
+
+## Considered Options
+
+* Leave as-is
+* Move endpoint in a new CDI version
+* Duplicate it in the standard path not taking CDI version into account
+* Add new endpoint on the standard path
+
+## Decision Outcome
+
+We should add a new endpoint under the standard path, but keep using the existing one internally:
```

Comment:
We could still use the old path internally for consistency, but it's not strictly necessary.
Backward compatibility is not an issue because we check the CDI versions.